core/basejob.py

```diff
@@ -493,7 +493,7 @@ def load_external(cls, path, settings=None, molecule=None, finalize=False):
             raise FileError('Path {} does not exist, cannot load from it.'.format(path))
 
         path = os.path.abspath(path)
-        jobname = os.path.basename(path)
+        jobname = jobname or os.path.basename(path)
```

Comment:
> Loading jobs that have been manually run often requires renaming files to fit the PLAMS conventions. By introducing this small change, at least the folder can be named differently than the files.

> I know that this can break some things, since the job is then not in the proper place. But I think if someone finds this option they know what they are doing...

I think this would then require a comment above this line or a mention in the docstring of the function that explains why you would use this argument and that it might have unintended consequences.

Reply:
Added a docstring.